package.json

```diff
@@ -7,19 +7,19 @@
     "bugs": "https://github.com/semaphore-protocol/semaphore/issues",
     "private": true,
     "scripts": {
-        "build": "yarn build:libraries && yarn build:subgraph",
-        "build:libraries": "yarn workspaces foreach -A -t --no-private run build",
-        "build:subgraph": "yarn workspace semaphore-subgraph codegen sepolia && yarn workspace semaphore-subgraph build",
+        "build": "yarn workspaces foreach -Ap --topological-dev run build",
```

Comment:
This is not just a perf change.
This will run ALL the build scripts in the monorepo (including e.g the website etc...).